test/helper/e2e/k8s/operator.go

```diff
@@ -47,162 +30,43 @@ var (
 )
 
 func BuildManager(initCfg *Config) (manager.Manager, error) {
-	scheme := runtime.NewScheme()
-	setupLog := ctrl.Log.WithName("setup")
+	akoScheme := runtime.NewScheme()
+	utilruntime.Must(scheme.AddToScheme(akoScheme))
+	utilruntime.Must(akov2.AddToScheme(akoScheme))
```

Comment:
We will have to move this away from here. But we do not need to do it here.
See CLOUDP-251767 but basically we want this add to scheme to be done once at the test suite setup. Otherwise you cannot launch several managers with different configurations, as they will race with each other.

Reply:
I actually gave a try, but everything has fallen apart and I decided to solve this in a separate PR